### PR TITLE
In support of modification of shoppingcart invoice email.

### PR DESCRIPTION
**Pull request by dcadams**
edx_platform PR:
https://github.com/Stanford-Online/edx-platform/pull/604

### CODE REVIEW

**Comment by dcadams** [comment]
@stvstnfrd

**Comment by stvstnfrd** [comment]
+1

File: playbooks/roles/edxapp/defaults/main.yml [hunk 1]
```diff
@@ -213,7 +213,9 @@ EDXAPP_ANALYTICS_API_KEY:  ""
 EDXAPP_LTI_USER_EMAIL_DOMAIN: "lti.example.com"
 # 900s, or 15 mins
 EDXAPP_LTI_AGGREGATE_SCORE_PASSBACK_DELAY: 900
+EDXAPP_PAYMENT_PLATFORM_NAME: 'your platform name'
```
Comment (stvstnfrd) [inline playbooks/roles/edxapp/defaults/main.yml]: Actually, shouldn't we just default this to "{ PLATFORM_NAME}" or whatever?

Comment (caesar2164) [inline playbooks/roles/edxapp/defaults/main.yml]: that seems like a sane default

Comment (stvstnfrd) [inline playbooks/roles/edxapp/defaults/main.yml]: Yeah, let's default this to
```yaml
EDXAPP_PAYMENT_PLATFORM_NAME: "{{ EDXAPP_PLATFORM_NAME }}"
```

**Pull request merged**